### DISCUSSION TITLE
Possibility to change the react prop value boolean value? (for ToggleSwitch button)

### DISCUSSION THREAD

**Discussion by simiboi184** [webdriverio/webdriverio#8426] [Community]
Google Chrome Extension ReactComponents:

props:
{
  "type": "switch",
  "className": "form-switch",
  "checked": false,
  "id": "toggle-switch",
}

![image](https://user-images.githubusercontent.com/19859202/173342851-70e44fc4-6a1c-4c7d-a9d9-4deae658af2a.png)

![image](https://user-images.githubusercontent.com/19859202/173342958-2fa3d0e1-cd4e-4b9e-9256-3a5ada4c98dc.png)

How do i use webdriver to click / toggle the switch button?

When i use $(("//input[@id="custom-switch"]).click() -> its says there's another element <label> on top that is being get clicked on. 

So i am looking at other options e.g., prop elements (from react component) wondering if there's possibility for us to change the value "checked": from false to true to simulate the "click" event?

**Comment by christian-bromann** [comment]
> its says there's another element on top that is being get clicked on.

Have you tried to click on the label instead? Can you identify the element that would receive the click? Note that in a real scenario this would happen to any user too.
